### PR TITLE
Feature: Homepage-EventCard component mobile responsive

### PR DESCRIPTION
**Pull request by SilviaTrujillano23**
## Describe your changes

- Update style in EventCard component 
- Responsive Events display for mobile (hide filters, headquarters dropdown and title)

## Issue ticket number and link
https://trello.com/c/0jJ2G57b

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update: 
- [x] Create storybook for components involved

**Events display: Mobile-version**
image

**Events display: Desktop-version**
image

### CODE REVIEW

File: public/programmingImg.png
Comment (jlora-mck) [inline public/programmingImg.png:1]: Please delete this file

**Review by jlora-mck — APPROVED** [review]
LGTM, left couple minor comments but its ok

**Review by jlora-mck — APPROVED** [review]

File: src/components/EventCard/EventCard.tsx [hunk 1]
```diff
@@ -1,167 +1,176 @@
...
+          <div className={classes.titleSection}>
+            <h2 className={classes.title}>{event.name}</h2>
+          </div>
+          <div className={classes.eventStatus}>
+            <ConferenceStatusSection status={event.status} />
+          </div>
+        </div>
+        <div className={classes.date}>
+          <h3 className={classes.day}>{getDatePart(event.eventDate)}</h3>
+        </div>
```
[the remark this answers is not in the document]
Comment (SilviaTrujillano23) [reply src/components/EventCard/EventCard.tsx]: Done, used Box and Typography components

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

**Pull request merged**